cmd/maya-apiserver/cstor-operator/handler.go

```diff
@@ -18,42 +18,39 @@ package spc
 
 import (
 	"fmt"
-	"strings"
-	"time"
-
 	"github.com/golang/glog"
-	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+	nodeselectv1alpha2 "github.com/openebs/maya/pkg/algorithm/caspool/v1alpha1"
+	apisv1alpha1 "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
 	openebs "github.com/openebs/maya/pkg/client/generated/clientset/versioned"
 	"github.com/pkg/errors"
 	k8serror "k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/runtime"
 	"k8s.io/client-go/tools/cache"
+	"time"
 )
 
-var (
-	// supportedPool is a map holding the supported raid configurations.
-	supportedPool = map[apis.CasPoolValString]bool{
-		apis.PoolTypeStripedCPV:  true,
-		apis.PoolTypeMirroredCPV: true,
-		apis.PoolTypeRaidzCPV:    true,
-		apis.PoolTypeRaidz2CPV:   true,
-	}
-)
-
-const (
-	// DiskStateActive is the active state of the disks.
-	DiskStateActive = "Active"
-	// ProvisioningTypeManual is the manual type of provisioning pool.
-	ProvisioningTypeManual = "manual"
-)
+// PoolOperation is used to create a cstor pool.
+type PoolOperation struct {
```

Comment:
Can we have this struct & related methods in separate file?

Reply:
tracking here 
https://github.com/openebs/maya/issues/1186